core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -81,3 +81,28 @@
     {% endif %}
   </ul>
 </script>
+<style type="text/css">
+.popover{
+  font-size: 13px;
+  max-width: 300px;
+  text-align: center;
+  z-index: -1;
+  color: white;
+  background-color: black;
+}
```

Comment:
@shivanXI 
I guess this code is almost the same as in mine [PR](https://github.com/oppia/oppia/pull/3267/files). With width being different. 
@anthkris  Would there be any conflicts merging both PRs due to duplicate work?